### DISCUSSION TITLE
Disable task list for certain issue templates

### DISCUSSION THREAD

**Discussion by muuvmuuv** [community/community#4319] [Projects and Issues]
Got that new task list feature and really liking it, but in issues like feature requests or bug issues they are often used to determine which part is involved. For example, in Angular: https://github.com/angular/angular-cli/issues/21202

Would be great to be able to disable this for some issue templates.

**Comment by denelon** [comment]
Another [example](https://github.com/microsoft/winget-pkgs/issues/18844) of how we use the checkbox not as a task list. Maybe we could get a "radio button" UI/UX for non tasks. https://github.com/github/feedback/discussions/4318

**Reply by dgw** [reply]
Checkboxes aren't even always tasks. We have a PR template that asks people to agree to the project license, confirm they've linted the code, etc. and all our PRs have "4 of 4 tasks completed" because of it.

Despite it not working for my org's use case, I do like the idea of a "radio button"—easy to come up with alternative syntax based on how the task list/checkbox feature already works: `- ( ) Radio button item`

**Comment by electronicboy** [comment]
I concur, I'd also like to add onto this as we use checkboxes as a "please confirm that you've done the following", it would be nice to be able to keep these checkboxes to make people to essentially "promise" us that they're meeting some requirements, e.g. they're actually using a supported version, but, the fact that they show up as tasks is a bit of a "bleh" aspect and makes them unviable to us, which would have been nice!

**Comment by pirate** [comment]
Sometimes I put mini multiple-choice surveys in issue comments and have people check the boxes to indicate their responses. With this kind of format the `3 of 7 Tasks` indicator doesn't make sense, it would be great to disable it for whole issue templates, or for arbitrary specific issues.

<img width="350" alt="Screen Shot 2021-07-14 at 12 31 25 a" src="https://user-images.githubusercontent.com/511499/125561609-3938947f-e261-4ac6-b407-8e46d28befdf.png">

**Reply by muuvmuuv** [reply]
Maybe we just need a option select toggle or something like that. I guess this is what the want us to do in futures using the new issue form builder instead of Markdown.

**Comment by DiegoCardoso** [comment]
We use checkboxes so the user can check which browsers are affected by the bug they are describing:

![image](https://user-images.githubusercontent.com/262432/125620182-3fc2cdbe-c966-4db3-9057-89d1cfc443e0.png)

**Reply by apiraino** [reply]
Another similar example of a checkbox list that is _not_ a task list: in the Rust project we use checkboxes to allow selecting which release channels are affected by an issue ([our template](https://github.com/rust-lang/rust/blob/master/.github/ISSUE_TEMPLATE/ice.yaml)), example:

![grafik](https://user-images.githubusercontent.com/6098822/215051278-75c7cbc4-1297-444f-8f7d-f5f83b8e8a9e.png)

**Comment by jcsp** [comment]
We hit this when trying to use checkboxes to identify which branches to backport issues to ([example](https://github.com/redpanda-data/redpanda/pull/5756))

Perhaps it would be sufficient to pick a different bracket character to mean "checkbox but it's not a todo list"

**Comment by thejeff77** [comment]
I'd really want a way to put multiple on a line  -- eg:

DONE / N/A
- [ ]  /  [ ]

So they'd have to tick "N/A" for stuff that isn't applicable. 

Or some fast way to cross out the whole line - the markup for that requires manual editing and its painful.

**Comment by muuvmuuv** [comment]
There is now an option to user a select box with options for that case I mentioned, just fyi. :)

**Reply by a-Leong** [reply]
Hello, can you explain what you mean by this? I have a use case much like the one you commented on above.

**Reply by muuvmuuv** [reply]
Here you can see it https://github.com/angular/angular/issues/new?assignees=&labels=&template=1-bug-report.yaml

**Reply by syed-a-ali** [reply]
Late to the party here but could I please see how the dropdown selection was implemented? I'm attempting to do the same for a pull request template. Attempt at Googling and viewing source code on the page did not bear fruit. Thanks in advance.

**Reply by muuvmuuv** [reply]
If the Angular example didn't work for you it might be that this feature is just for larger repo>/owners.
https://raw.githubusercontent.com/angular/angular/main/.github/ISSUE_TEMPLATE/1-bug-report.yaml

**Reply by syed-a-ali** [reply]
Thank you, I understand now, it seems the yaml template is only available for issue report which support these features. Pull request template only has markdown available presently.

**Comment by Khushboqof** [comment]
Sometimes I put mini multiple-choice surveys in issue comments and have people check the boxes to indicate their responses. With this kind of format the 3 of 7 Tasks indicator doesn't make sense, it would be great to disable it for whole issue templates, or for arbitrary specific issues.

**Comment by rewati-arista** [comment]
There is another hack which stops github from treating it as task list, which is just making these checkboxes as bullet points so doing:
- - [ ] Option 1
- - [ ]  Option 2 
this wont flag them as tasks and you can still select one option :)

**Reply by felipelima-circle** [reply]
The markdown for this would be something like:
```
- - [ ] Option 1
- - [ ] Option 2
```
